cli/tests/integrations/test_service.py

```diff
@@ -93,7 +93,7 @@ def test_service_inactive():
 
 
 def test_service_completed():
-    package_install('cassandra', True)
+    package_install('cassandra', True, ['--package-version=0.2.0-1'])
```

Comment:
Okay for now. Are we solving it this way because the latest Cassandra had too many changes? Let's file an issue to use the latest version.